core/src/plugins/filed/bareos-fd-ovirt.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+import sys
+import os.path
+
+libdirs = ["/usr/lib64/bareos/plugins/", "/usr/lib/bareos/plugins/"]
+sys.path.extend([l for l in libdirs if os.path.isdir(l)])
+
+# Provided by the Bareos FD Python plugin interface
+from bareos_fd_consts import bRCs
+
+# This module contains the wrapper functions called by the Bareos-FD, the
+# functions call the corresponding
+# methods from your plugin class
+import BareosFdWrapper
+from BareosFdWrapper import *  # noqa
```

Comment:
Correct, that bothered me, too. But it's a _really_ bad idea to remove that line, as python-fd.cc requires that the plugin provides the functions declared in BareosFdWrapper.py. I hope that this can be overhauled in the future, as the Python plugins need to be adapted to Python 3 anyway.
Meanwhile, I've added the function names to avoid the "import *".